### PR TITLE
Add basic page layout with EUI

### PR DESCRIPTION
**Pull request by jonathonherbert**
## What does this change?

Adds a basic page layout with EUI. Behold:

Screenshot 2023-03-20 at 17 31 47

### Dev notes

Behold, but less triumphantly (this is a production build running in CODE):

Screenshot 2023-03-20 at 17 59 44

... that'll be the next PR 😅. Other tools that use EUI in production come in at about 500kb, so this is likely a problem with our build. Build times are also appalling – I suspect a move to Vite may kill both birds with one stone.

Other notes: I've added a basic CSS reset, taking a few things from [here.](https://www.joshwcomeau.com/css/custom-css-reset/) There's also a CSS override to ensure that our headings are Guardian-specific. I thought this was more elegant than custom heading components for each heading level, but we could move this to heading components if others felt strongly.

## How to test

Run locally, or in CODE, remembering to run `npm i` within rule-manager-client to refresh dependencies – `@guardian/source` is out, `@elastic/eui` is in.

### CODE REVIEW

**Review by phillipbarron — APPROVED** [review]
Looks great, Jon. Is Elastic UI the library that's preventing us using React 18?

File: apps/rule-manager/rule-manager-client/package.json [hunk 1]
```diff
@@ -15,27 +15,29 @@
...
-    "react-dom": "^16.14.0"
+    "@elastic/datemath": "^5.0.3",
+    "@elastic/eui": "^76.2.0",
+    "@emotion/css": "^11.10.6",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "moment": "^2.29.4",
+    "prop-types": "^15.8.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
```
[the remark this answers is not in the document]
Comment (jonathonherbert) [reply apps/rule-manager/rule-manager-client/package.json]: Thanks @mxdvl! [f068c9e](https://github.com/guardian/typerighter/pull/277/commits/f068c9ecfb4c324bcb3affeeb2f3534fa694c0bc)

**Comment by jonathonherbert** [comment]
That's right, https://github.com/elastic/eui/issues/6182 is tracking that issue. The list of issues is small, so hopefully it's not too far off!

File: apps/rule-manager/rule-manager-client/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonathonherbert) [reply apps/rule-manager/rule-manager-client/package.json]: https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

**Pull request merged**